classes/plugins/load.py

```diff
@@ -0,0 +1,75 @@
+from multiprocessing import Array
+from . import api
+from . import event
+
+import os
+import logging
+import inspect
+
+def _getPlugins_() -> Array:
+    import pkgutil
+    import importlib
+    if not (os.path.exists('plugins/')):
+        os.mkdir('plugins/')
+    import plugins as exts
+
+    def unqualify(name: str) -> str:
+        """Return an unqualified name given a qualified module/package `name`."""
+        return name.rsplit(".", maxsplit=1)[-1]
+    plugins = {}
+    for module in pkgutil.walk_packages(exts.__path__, f"{exts.__name__}."):
+        plugin_name = unqualify(module.name)
+        if plugin_name.startswith("_"):
+            next
+        plugins[plugin_name] = importlib.import_module(module.name)
+    order = {}
+    for plugin_name in plugins:
+        plugin = plugins[plugin_name]
+        try:
+            tmp = order[int(plugin.PRIORITY)]
+        except KeyError:
+            order[int(plugin.PRIORITY)] = []
+        except AttributeError:
+            plugin.PRIORITY = 999
+        order[int(plugin.PRIORITY)].append(plugin)
+    max = 0
+    for p in order:
+        plug = order[p]
+        for i in plug:
+            if int(i.PRIORITY) > max:
+                max = int(i.PRIORITY)
+    ret = []
+    for i in range(max+1):
+        ret.append(None)
```

Comment:
You shouldn't fill the entire list with `None` values, the list of items you return should only contain the items of interest. What I would recommend doing is flattening out whatever's inside your plugins (handlers I assume? is what you're iterating over) into a single dictionary, probably with priority as keys and a simple list of all handlers with that priority. Then it would be easier to return a list of all handlers, in order, without filling a list with an indeterminate (and possible infinite) number of empty items

Reply:
i was planning on implementing that where it iterates over the list and converts the list of list into a single list removing the `None` types

Reply:
done
also i should implement a `MAX_PRIORITY` and `DEFAULT_PRIORITY` option in config
that or a optional flag to change it

Reply:
ex: --max-priority [limit] and --default-priority [default]
and also set the default maximum to 1000 and the default to 999 (incase there are any plugins that want to load last)